pkg/controller/rbac/provider/binding/reconciler.go

```diff
@@ -67,8 +65,6 @@ func Setup(mgr ctrl.Manager, log logging.Logger) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		Named(name).
 		For(&v1alpha1.ProviderRevision{}).
-		Owns(&rbacv1.ClusterRoleBinding{}).
-		Watches(&source.Kind{Type: &corev1.ServiceAccount{}}, &handler.EnqueueRequestForOwner{OwnerType: &v1alpha1.ProviderRevision{}}).
```

Comment:
This seems like it reverts https://github.com/crossplane/crossplane/pull/1919, which was put in place to address a bug that I observed. Do we have a reason to believe we won't see that bug again?

Reply:
No I think that could occur again. We should be able to add back the requeue safely for SA as it is not watched by the Namespace controller. Does that sound good to you?